config/routes.rb

```diff
@@ -0,0 +1,73 @@
+Rails.application.routes.draw do
+
+root 'tasks#index'
+get '/' => 'tasks#index'
+get '/tasks/new' => 'tasks#new'
+post '/tasks' => 'tasks#create', as: 'tasks'
+delete '/tasks/:id' => 'tasks#delete'
+get 'tasks/:id' => 'tasks#show', as: 'task'
+get 'tasks/:id/edit' => 'tasks#edit', as: 'edit_task'
+patch 'tasks/:id' => 'tasks#update'
+
+
+get '/people'=> 'people#index'
+get 'people/:id' => 'people#show', as: 'person'
+get 'people/:id/tasks' => 'people#tasks', as: 'person_task'
+
+#resources :tasks
```

Comment:
I like to clear out all this extra commented out stuff in some files, which you can totally do if you want. Fine to leave it too!